<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DrumPadWidget</class>
 <widget class="QSplitter" name="DrumPadWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>525</height>
   </rect>
  </property>
  <property name="orientation">
   <enum>Qt::Horizontal</enum>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <layout class="QVBoxLayout">
    <property name="topMargin">
     <number>9</number>
    </property>
    <item>
     <layout class="QVBoxLayout">
      <item>
       <layout class="QHBoxLayout" stretch="1,0,0,0">
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MidiButton" name="pushButtonUp">
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>⬆</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MidiButton" name="pushButtonDown">
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>⬇</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonRefresh">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>↻</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTreeView" name="presetsView">
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="DrumPadPresetDetailWidget" name="presetDetailWidget">
       <attribute name="title">
        <string>Properties</string>
       </attribute>
      </widget>
      <widget class="QTreeView" name="filesView">
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <attribute name="title">
        <string>Samples</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="SequencerWidget" name="sequencerWidget" native="true"/>
   <widget class="DrumPadSamplesWidget" name="samplesWidget" native="true"/>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MidiButton</class>
   <extends>QPushButton</extends>
   <header>widgets/midibutton.h</header>
  </customwidget>
  <customwidget>
   <class>DrumPadSamplesWidget</class>
   <extends>QWidget</extends>
   <header>widgets/drumpadsampleswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DrumPadPresetDetailWidget</class>
   <extends>QScrollArea</extends>
   <header>widgets/drumpadpresetdetailwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SequencerWidget</class>
   <extends>QWidget</extends>
   <header>widgets/sequencerwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
